prog/postgres/postgres_resource_nexus.rb

```diff
@@ -49,10 +49,7 @@ def self.assemble(project_id:, location:, name:, target_vm_size:, target_storage
 
       PostgresFirewallRule.create_with_id(postgres_resource_id: postgres_resource.id, cidr: "0.0.0.0/0")
 
-      primary = Prog::Postgres::PostgresServerNexus.assemble(resource_id: postgres_resource.id, timeline_id: timeline_id, timeline_access: timeline_access, representative_at: Time.now).subject
-      postgres_resource.required_standby_count.times do
-        Prog::Postgres::PostgresServerNexus.assemble(resource_id: postgres_resource.id, timeline_id: timeline_id, timeline_access: "fetch", private_subnet_id: primary.vm.private_subnets.first.id)
-      end
+      Prog::Postgres::PostgresServerNexus.assemble(resource_id: postgres_resource.id, timeline_id: timeline_id, timeline_access: timeline_access, representative_at: Time.now).subject
```

Comment:
nit: no need to subject
```suggestion
      Prog::Postgres::PostgresServerNexus.assemble(resource_id: postgres_resource.id, timeline_id: timeline_id, timeline_access: timeline_access, representative_at: Time.now)
```